Java/core/src/main/java/com/amazon/randomcutforest/RandomCutForest.java

```diff
@@ -484,6 +484,17 @@ public void setBoundingBoxCacheFraction(double cacheFraction) {
         updateExecutor.forEachTree(t -> t.setBoundingBoxCacheFraction(cacheFraction));
     }
 
+    /**
+     * changes the setting of time dependent sampling on the fly
+     * 
+     * @param lambda new value of sampling rate
+     */
+
```

Comment:
minor. no need for blank line

Reply:
fixed

Reply:
the change is not in the current version

Reply:
Should be fixed now.